providers/azureblob/src/main/java/org/jclouds/azureblob/blobstore/AzureBlobRequestSigner.java

```diff
@@ -51,19 +52,24 @@
    private final Provider<String> timeStampProvider;
    private final DateService dateService;
    private final SharedKeyLiteAuthentication auth;
+   private final String credential;
+   private final boolean isSAS; 
+  
 
    @Inject
    public AzureBlobRequestSigner(
          BlobToHttpGetOptions blob2HttpGetOptions, @TimeStamp Provider<String> timeStampProvider,
          DateService dateService, SharedKeyLiteAuthentication auth,
-         @org.jclouds.location.Provider Supplier<Credentials> creds)
+         @org.jclouds.location.Provider Supplier<Credentials> creds, @Named("sasAuth") boolean sasAuthentication)
          throws SecurityException, NoSuchMethodException {
       this.identity = creds.get().identity;
+	  this.credential = creds.get().credential;
       this.storageUrl = URI.create("https://" + creds.get().identity + ".blob.core.windows.net/");
       this.blob2HttpGetOptions = checkNotNull(blob2HttpGetOptions, "blob2HttpGetOptions");
       this.timeStampProvider = checkNotNull(timeStampProvider, "timeStampProvider");
       this.dateService = checkNotNull(dateService, "dateService");
       this.auth = auth;
+	  this.isSAS = sasAuthentication;
```

Comment:
Tabs instead of spaces (repeated throughout).

Reply:
sorry,  I hope I've now managed to identify all places with tabs

Reply:
it really looked weird in git.